### ISSUE TITLE
GoldenDict does not show examples lines (with SOLUTION)

### ISSUE THREAD

**Issue by RadzionA** [goldendict/goldendict#1084]
Problem: Couldn't see examples for words usage in GoldenDict (very powerful and popular cross-platform dictionary). I have met this problem on Linux Mint, but it can be cross-platform.

Problem core: in the source code of the articles pages I have found that tags are used as block elements without correct style formatting, so the parser doesn't show them on the page - text in the source code, but not visible.

SOLUTION:

1) On FAQ page http://goldendict.org/wiki/index.php/FAQ I found how to add custom styles for the article pages:

    The article text you see is actually an HTML page. It has its own CSS style which can be arbitrarily altered. To do so, you need to create a special text file article-style.css, located in Linux in ~/.goldendict, and in Windows in %APPDATA%\GoldenDict. You can put arbitrary CSS code there.

So I created such style file in the path provided.

2) I added this style to parse span blocks correctly (the way they are used in the page code):

.dsl_ex,.dsl_opt{
    display: inline-block;
}

After this, you should open an article again and see that the problem is solved. 

Example of how the article looks with the error and after fixing styles: https://ibb.co/jTHesA

Also created the article on superuser.com:
https://superuser.com/questions/1377930/goldendict-does-not-show-examples-lines

**Comment by RadzionA** [comment]
vedgy, thanks, I now checked that solution and it also works.
But in general, such program behavior is definitely wrong: if examples are collapsed I shouldn't see any examples at all, but I see just list bullets without text...

**Comment by vedgy** [comment]
Indeed. This collapsing of examples should save space and make articles more compact to be useful, but apparently it doesn't, at least for some dictionaries. I wonder if anyone actually prefers such collapsed display. If most users don't, perhaps the *Expand optional parts* should be checked by default.

**Comment by xfeep** [comment]
The Expand optional parts really should be checked by default.
We waste too much time to find this solution to resolve this issue.

**Comment by gavenkoa** [comment]
Newer versions have "Preference" => "Advanced" => "Expand optional parts". No more CSS hacks.

Similar: #1065  & #1074.
